### PR TITLE
Updated ADOPTERS.md for Power BI

### PR DESCRIPTION
**Pull request by CodeCyclone**
# PR Summary

Added a mention for Power BI's adaption of PowerShell to ADOPTERS.md.

## PR Context

Raise awareness of Power BI's usage of PowerShell and how critical it is to this service both externally and internally.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -34,4 +34,6 @@ This is a list of adopters of using PowerShell in production or in their product
 * [GitHub Actions Virtual Environments for Hosted Runners](https://help.github.com/actions/reference/virtual-environments-for-github-hosted-runners) Windows, Ubuntu, and macOS virtual environments used by customers of GitHub Actions include PowerShell out of the box.
 * [GitHub Actions Python builds](https://github.com/actions/python-versions) GitHub Actions uses PowerShell to automate building Python from source for its runners.
 * [Microsoft HoloLens](https://www.microsoft.com/hololens) makes extensive use of PowerShell 7+ throughout the development cycle to automate tasks such as firmware assembly and automated testing.
+* [Power BI](https://powerbi.microsoft.com/) provides PowerShell users a set of cmdlets in [MicrosoftPowerBIMgmt](https://docs.microsoft.com/en-us/powershell/power-bi/overview?view=powerbi-ps) module to manage and automate the Power BI service.
```
Comment (SteveL-MSFT) [inline ADOPTERS.md:37]:
```suggestion
* [Power BI](https://powerbi.microsoft.com/) provides PowerShell users a set of cmdlets in [MicrosoftPowerBIMgmt](https://docs.microsoft.com/powershell/power-bi/overview) module to manage and automate the Power BI service.
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
URL should not have a specific locale nor view as it will adapt automatically to the user

**Comment by SteveL-MSFT** [comment]
Looks like you'll also need to update the `.spelling` file to include the word `cmdlets`

File: ADOPTERS.md (hunk 1 above)
Comment (CodeCyclone) [reply ADOPTERS.md]: Fixed in commit 4cee5a4d065a4edc8c0275f46f46ba6aaaf6fe3e

**Comment by CodeCyclone** [comment]
> Looks like you'll also need to update the `.spelling` file to include the word `cmdlets`

Looks like it is already there on line 140

**Comment by SteveL-MSFT** [comment]
@CodeCyclone must be complaining about the spelling of `MicrosoftPowerBIMgmt` then

**Review by SteveL-MSFT — APPROVED** [review]

**Review by daxian-dbw — CHANGES_REQUESTED** [review]
@CodeCyclone You need to add the following to the end of the `.spelling` file:
```
 - ADOPTERS.md
MicrosoftPowerBIMgmt
```

**Comment by daxian-dbw** [comment]
Superseded by #17766

**Pull request closed without merging**